css/DayPicker.scss

```diff
@@ -29,6 +29,10 @@
   color: $react-dates-color-placeholder-text;
   cursor: pointer;
   line-height: 0.78;
+  -moz-user-select: none;
+  -webkit-user-select: none;
+  -ms-user-select: none;
```

Comment:
Airbnb is - whether someone uses it or not seems like the top-level app's choice to make.